index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles.css" /> 
+    <script src="https://kit.fontawesome.com/26c38390b4.js" crossorigin="anonymous"></script>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <div class="background"><img src="./headline-mobile.png"></div>
+    <div class="top">
+        <div class="header">
+            <p class="logo">My Logo</p>
+        </div>
+        <div class="menuhover">
+            <button>
+                 <i class="fa-solid fa-bars"></i>
+            </button>
+        </div>
+    </div>
```

Comment:
- [ ] [OPTIONAL] The container of this toolbar section is a header tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

- [ ] [OPTIONAL] The logo has a link (`a` tag). * This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

**_I am marking this comment as optional because the previous reviewer did not mention it. But I strongly suggest you fix these issues as they are core requirements that the previous reviewer missed._**